airflow/www/views.py

```diff
@@ -3456,10 +3456,7 @@ def grid_data(self):
             if run_state:
                 query = query.filter(DagRun.state == run_state)
 
-            ordering = (DagRun.__table__.columns[name].desc() for name in dag.timetable.run_ordering)
-            dag_runs = query.order_by(*ordering, DagRun.id.desc()).limit(num_runs).all()
-            dag_runs.reverse()
-
+            dag_runs = wwwutils.sorted_dag_runs(query, ordering=dag.timetable.run_ordering, limit=num_runs)
```

Comment:
This feels more like it should be a method on DagRun class maybe? (That way this can be used by API or CLI too)

Reply:
This function in general includes some UI-specific sorting/slicing logic I don’t feel right making generic. The inner `_get_run_ordering_expr` probably should, but I’d rather wait until when we actually reuse it somewhere (we don’t right now).